openmctStaticServer/walkingCtrlPortDataStruct.json

```diff
@@ -0,0 +1,166 @@
+  "value.left_foot::angular_velocity::measured.0": 0,
+  "value.left_foot::angular_velocity::measured.1": 0,
+  "value.left_foot::angular_velocity::measured.2": 0,
+  "value.left_foot::linear_force::measured.0": 37.77941563755032,
+  "value.left_foot::linear_force::measured.1": -11.901542041021923,
+  "value.left_foot::linear_force::measured.2": 154.6893735555257,
+  "value.left_foot::linear_velocity::measured.0": 0,
+  "value.left_foot::linear_velocity::measured.1": 0,
+  "value.left_foot::linear_velocity::measured.2": 0,
+  "value.left_foot::orientation::desired.0": 0,
+  "value.left_foot::orientation::desired.1": 0,
+  "value.left_foot::orientation::desired.2": 0,
+  "value.left_foot::orientation::measured.0": 0.0003857060242884855,
+  "value.left_foot::orientation::measured.1": -0.0013937181616784772,
+  "value.left_foot::orientation::measured.2": -0.000866652398915334,
+  "value.left_foot::position::desired.0": 0.9878787878788007,
+  "value.left_foot::position::desired.1": 0.08,
+  "value.left_foot::position::desired.2": 0,
+  "value.left_foot::position::measured.0": 0.9882652502842438,
+  "value.left_foot::position::measured.1": 0.08044679903060947,
+  "value.left_foot::position::measured.2": -0.000021531704383528916,
+  "value.right_foot::angular_torque::measured.0": 0.08864232148801479,
+  "value.right_foot::angular_torque::measured.1": -12.550430384408886,
+  "value.right_foot::angular_torque::measured.2": 5.313339776821795,
+  "value.right_foot::angular_velocity::measured.0": 0,
+  "value.right_foot::angular_velocity::measured.1": 0,
+  "value.right_foot::angular_velocity::measured.2": 0,
+  "value.right_foot::linear_force::measured.0": -37.09692146763135,
+  "value.right_foot::linear_force::measured.1": 11.03154155370319,
+  "value.right_foot::linear_force::measured.2": 169.50148453275494,
+  "value.right_foot::linear_velocity::measured.0": 0,
+  "value.right_foot::linear_velocity::measured.1": 0,
+  "value.right_foot::linear_velocity::measured.2": 0,
+  "value.right_foot::orientation::desired.0": 0,
+  "value.right_foot::orientation::desired.1": 0,
+  "value.right_foot::orientation::desired.2": 0,
+  "value.right_foot::orientation::measured.0": 0,
+  "value.right_foot::orientation::measured.1": 0,
+  "value.right_foot::orientation::measured.2": 0,
+  "value.right_foot::position::desired.0": 0.9878787878788007,
+  "value.right_foot::position::desired.1": -0.08,
+  "value.right_foot::position::desired.2": 0,
+  "value.right_foot::position::measured.0": 0.9878787878788007,
+  "value.right_foot::position::measured.1": -0.08,
+  "value.right_foot::position::measured.2": 0,
+  "value.zmp::desired.0": 1.0180441241050355,
+  "value.zmp::desired.1": -0.00006235474681043815,
+  "value.zmp::desired_planner.0": 1.0178787878788007,
+  "value.zmp::desired_planner.1": 0.00011546157815479083,
+  "value.zmp::measured.0": 1.0180768436588523,
+  "value.zmp::measured.1": 0.0008882254867355402
+}
```

Comment:
Is this file used? It seems the only file that encodes a priori a structure in the data published in the `BipedalLocomotion.YarpUtilities.VectorsCollection`, that may be problematic if the walking-controller is called with a slightly different configuration.

Reply:
No sorry, this is a left-over. It was used initially, for test purposes, before I implemented the telemetry sample request fetching the reference sample to parse the info from. I'll just remove it.

Reply:
Fixed